department-of-veterans-affairs/abd-vro
# WIP: Add Shellcheck to Pre-commit and Batch fix issues

## What was the problem?

We had many warnings and improvements for implementing best practices in our shell scripts.

## How does this fix it?[^1]

In order to ensure strong code quality going forward, I added the shellcheck tool to run as a pre-commit hook. I also ran pre-commit on all files in the repo (except those explicitly excluded) to get us to a clean state with our pre-commit tool.

## How to test this PR
IN PROGRESS

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
